src/components/Camera/index.js

```diff
@@ -65,8 +52,8 @@ export class CameraPure extends React.Component<CameraPureType> {
   }
 
   render() {
-    const {method, title, subTitle, onUploadFallback, onFallbackClick, hasError,
-      onUserMedia, onFailure, webcamRef, isFullScreen, i18n, video} = this.props;
+    const {method, title, subTitle, hasError, onUserMedia,
```

Comment:
the `onUploadFallback` is coming from the `capture` component. Logic there should also be removed.

Reply:
I'm going to need that logic in this PR #372 which is pointing at this branch